### PR TITLE
Optimize name rules

### PR DESCRIPTION
**Pull request by anstns**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Update slice creation method and name specification

#### Which issue(s) this PR fixes:

Fixes #
none
#### Special notes for your reviewer:
Thank you
#### Does this PR introduce a user-facing change?

```release-notes
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
hi. you'd have to sign the cla before this pr is accepted.

**Comment by anstns** [comment]
> /easycla

Has been successfully signed

**Comment by anstns** [comment]
lgtm

File: cmd/kubeadm/app/images/images.go [hunk 1]
```diff
@@ -88,26 +88,26 @@ func GetEtcdImage(cfg *kubeadmapi.ClusterConfiguration) string {
 
 // GetControlPlaneImages returns a list of container images kubeadm expects to use on a control plane node
 func GetControlPlaneImages(cfg *kubeadmapi.ClusterConfiguration) []string {
-	imgs := []string{}
+	image := make([]string, 0)
```
Comment (neolit123) [inline cmd/kubeadm/app/images/images.go:91]: you can omit the imgs -> image rename.

Comment (anstns) [reply cmd/kubeadm/app/images/images.go:91]: I feel like it would be more intuitive to write

Comment (neolit123) [reply cmd/kubeadm/app/images/images.go:91]: please use 'images' in that case

Comment (neolit123) [reply cmd/kubeadm/app/images/images.go:91]: also keep the commits squashed to 1.

Comment (anstns) [reply cmd/kubeadm/app/images/images.go]: I have finished fixing it, does it meet the requirements

**Comment by anstns** [comment]
> /ok-to-test /lgtm /approve

Thank you for your recognition, this is the first time in my life

**Pull request merged**